### ISSUE TITLE
Error after upgrading to 5.3.0

### ISSUE THREAD

**Issue by pbartyik** [firefly-iii/firefly-iii#3518]
I am running Firefly III version 5.2.8.

**Description**

I've upgraded from 5.2.8 to 5.3.0 as per [instructions](https://docs.firefly-iii.org/advanced-installation/upgrade#virtual-or-real-server).
I've installed php7.4 packages corresponding to currently installed 7.3 ones: `aptitude search php7.3 | grep ^i | cut -d ' ' -f 3 | sed -r 's/3/4/g' | xargs sudo apt install -y`
After navigating to site I'm greeted with the error `syntax error, unexpected 'string' (T_STRING), expecting function (T_FUNCTION) or const (T_CONST)`

I've tried to reinstall composer via apt as well and repeat the upgrade, didn't help.

**Extra info**
 
Running Linux sol 4.15.0-109-generic #110-Ubuntu SMP Tue Jun 23 02:39:32 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
/debug also throws the same error, can't share

**Bonus points**

>cat ff3-cli-2020-07-03.log 
[2020-07-03 08:54:38] local.WARNING: Database entry exists. Update to latest version (15)  
[2020-07-03 08:54:46] local.WARNING: Database entry exists. Update to latest version (15) 
- I have read the FAQ at https://bit.ly/FF3-FAQ
- I added log files

**Comment by JC5** [comment]
Could it be that your webserver is still loading PHP7.3 packages? That sometimes happens. Try creating the file `public/pi.php` next to `public/index.php` and add the following content:

```php
<?php
phpinfo();
```

It should report back PHP 7.4

**Comment by pbartyik** [comment]
Yep. In one of my available nginx blocks I've set default to 7.3, changed to 7.4 works like a charm.
```
upstream php-handler {
        server unix:/run/php/php7.4-fpm.sock;
}
```

**Comment by JC5** [comment]
:+1:
